### PR TITLE
Last two sections of About page

### PR DESCRIPTION
**Pull request by seanbsullivan**

### CODE REVIEW

**Review by wcrum — APPROVED** [review]
Looks good to me!

File: src/pages/about.astro [hunk 1]
```diff
@@ -96,6 +111,44 @@ import FooterFull from '../components/footer/FooterFull.svelte'
...
+        </div>
+        <div class="font-light text-white sm:text-lg">
+          <div class="flex justify-center">
+            <img
+              src="icons/cta.svg"
+              class="md:h-[107px] md:w-[109px] md:min-w-[109px]"
+            />
+          </div>
+          <h2
+            class="mb-4 pt-8 mt-8 leading-normal lg:pt-0 text-xl md:text-[40px] font-extrabold tracking-tight text-center text-mcswf-gold"
```
Comment (wcrum) [inline src/pages/about.astro]: Probably should either be pt-8 or mt-8 not both.

File: src/pages/about.astro [hunk 2]
```diff
@@ -96,6 +111,44 @@ import FooterFull from '../components/footer/FooterFull.svelte'
...
+          <img
+            class="w-3/4 rounded-lg aspect-square object-cover md:h-[550px] md:w-[550px] md:min-w-[550px]"
+            src="images/mcswf_marine_sgt.png"
+          />
+        </div>
+        <div class="font-light text-white sm:text-lg">
+          <div class="flex justify-center">
+            <img
+              src="icons/cta.svg"
+              class="md:h-[107px] md:w-[109px] md:min-w-[109px]"
```
Comment (wcrum) [inline src/pages/about.astro]: Can you use only base and lg: breakpoint, could replace w- and h- with `h-28` `w-28`

File: src/pages/about.astro [hunk 3]
```diff
@@ -96,6 +111,44 @@ import FooterFull from '../components/footer/FooterFull.svelte'
...
+        class="items-center max-w-screen-xl gap-16 px-4 py-8 mx-auto lg:grid lg:grid-cols-2 md:py-8 md:px-6"
+      >
+        <div class="flex justify-center">
+          <img
+            class="w-3/4 rounded-lg aspect-square object-cover md:h-[550px] md:w-[550px] md:min-w-[550px]"
+            src="images/mcswf_marine_sgt.png"
+          />
+        </div>
+        <div class="font-light text-white sm:text-lg">
+          <div class="flex justify-center">
```
Comment (wcrum) [inline src/pages/about.astro:119]: Should probably add padding.

**Review by wcrum — CHANGES_REQUESTED** [review]
Tailwind classes and padding.

File: src/pages/about.astro (hunk 3 above)
Comment (seanbsullivan) [reply src/pages/about.astro:119]: I thought the spacing looked good, not sure how it'll look once the metric cards have borders but maybe we can merge this and then make that change? or if you think the spacing is off just let me know how much and I can adjust

File: src/pages/about.astro (hunk 2 above)
Comment (seanbsullivan) [reply src/pages/about.astro]: will do

File: src/components/buttons/WhiteButton.svelte [hunk 4]
```diff
@@ -2,12 +2,13 @@
   import Icon from '@iconify/svelte'
   export let text
   export let href
-  export let icon
-  export let extras
+  // these don't seem to be used for anything
```
Comment (dfullaway) [inline src/components/buttons/WhiteButton.svelte:5]: I suggest deleting these instead of just commenting them out.

File: src/pages/about.astro [hunk 5]
```diff
@@ -1,15 +1,30 @@
...
+// interface Props {
+//   entry: CollectionEntry<'aboutMetrics'>
+// }
 
 import '../styles/global.css'
-import { imageConfig } from 'astro:assets'
-import ContentImageFull from '../components/sections/ContentImageFull.svelte'
 
-import FooterFull from '../components/footer/FooterFull.svelte'
+// export async function getStaticPaths() {
```
Comment (dfullaway) [inline src/pages/about.astro]: Recommend removing the commented out sections - it will just cause confusion later.

Comment (seanbsullivan) [reply src/pages/about.astro]: good call thanks will do

**Pull request merged**